kcidb/monitor/subscriptions/linux_stable_rc.py

```diff
@@ -19,6 +19,14 @@ def match_revision(revision):
     if revision.builds_valid is None:
         return ()
 
+    # If the revision is not from 'maestro' or 'broonie' origin
+    origins = []
+    for _, branch_checkouts in revision.repo_branch_checkouts.items():
+        for _, checkouts in branch_checkouts.items():
+            origins.extend([checkout.origin for checkout in checkouts])
+    if origins not in ('maestro', 'broonie'):
+        return ()
```

Comment:
This will never be true, because an array of anything will never be found in `('maestro', 'broonie')`.

Reply:
Yes, I realized that from `pytest` results. Fixed it.